projects/pt1/e2e_testing/xfail_sets.py

```diff
@@ -399,6 +399,7 @@
     "AtenIntBoolOpModule_basic",
     "AtenIntMM_basic",
     "AtenItemFpOpModule_basic",
+    "AtenNonzero1DDynamicModule_basic",
```

Comment:
What's the error in case of `fx_importer` config? Since you have added a decomposition (and if that's correct?) then ideally the test should have passed for this config.

Reply:
```
    FAIL - "AtenNonzero1DDynamicModule_basic"
        Runtime error: Traceback (most recent call last):
          File "/proj/gdba/shark/chi/src/torch-mlir/build/tools/torch-mlir/python_packages/torch_mlir/torch_mlir_e2e_test/framework.py", line 354, in compile_and_run_test
            trace = config.run(compiled, golden_trace)
          File "/proj/gdba/shark/chi/src/torch-mlir/build/tools/torch-mlir/python_packages/torch_mlir/torch_mlir_e2e_test/configs/fx_importer_backend.py", line 49, in run
            self._export_run(artifact, trace)
          File "/proj/gdba/shark/chi/src/torch-mlir/build/tools/torch-mlir/python_packages/torch_mlir/torch_mlir_e2e_test/configs/fx_importer_backend.py", line 124, in _export_run
            module = fx.export_and_import(
          File "/proj/gdba/shark/chi/src/torch-mlir/build/tools/torch-mlir/python_packages/torch_mlir/torch_mlir/fx.py", line 100, in export_and_import
            return _module_lowering(
          File "/proj/gdba/shark/chi/src/torch-mlir/build/tools/torch-mlir/python_packages/torch_mlir/torch_mlir/fx.py", line 42, in _module_lowering
            run_pipeline_with_repro_report(
          File "/proj/gdba/shark/chi/src/torch-mlir/build/tools/torch-mlir/python_packages/torch_mlir/torch_mlir/compiler_utils.py", line 127, in run_pipeline_with_repro_report
            raise TorchMlirCompilerError(trimmed_message) from None
        torch_mlir.compiler_utils.TorchMlirCompilerError: Lowering TorchFX IR -> Torch Backend IR failed with the following diagnostics:
```

Reply:
more detail with ` torch-mlir-opt -pass-pipeline='builtin.module(func.func(torch-match-quantized-custom-ops), torchdynamo-export-to-torch-backend-pipeline{extra-library=})' /tmp/UnnammedModule.mlir --debug`

https://gist.github.com/AmosLewis/d9a0193b5600139836c331f6bfa19e98

Reply:
@AmosLewis do we have a pattern/lowering for `torch.aten.sym_constrain_range_for_size` somewhere?

Reply:
and the e2e tests for onnx pass, so the decomposition is definitly correct.